hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,37 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset in state, this field MUST NOT be set.<br/>
+     * If unset in this transaction and set in state, the current value SHALL
+     *  be removed.<br/>
```

Comment:
We may need a wrapper here too, we shouldn't require the `fee_schedule_key` to be set in every update transaction just to avoid _clearing_ it.
Alternatively we can behave like `token_update`, and declare that setting a "role" key to an empty `KeyList` key value removes that "role" key.